README.md

```diff
@@ -31,10 +31,12 @@ After completing this code pattern, the user will learn how to:
 
 ![](doc/source/images/architecture.png)
 
+1. A model is created in Watson Knowledge Studio using corpus and type files.
 1. The user provides the client with an SMS message to be analyzed.
-2. The client sends the SMS to Watson NLU for analysis, specifying which machine learning WKS model to use.
-3. Watson NLU extracts the domain specific entities and returns the results to the client.
-4. The client renders the results to the user.
+1. The client sends the SMS to Watson NLU for analysis
+1. Watson NLU loads the WKS model.
+1. Watson Knowledge Studio extracts domain specific entities and sends the results back to Watson NLU.
+1. Watson NLU returns the results to the client and the user.
 
 ### How does Watson Knowledge Studio work?
 
```

Comment:
So I might do an image that says:
(trainer user?->) JSON 1. -> WKS
WKS (model) 2 -> NLU
So that's the train and deploy part, and then
User 3 -> Browser (sms text) 4 -> NLU
NLU 5 -> Browser 6? -> User

Some of the "User" details and numbers going both ways might not be needed, but I think you get the idea.  The current pic looks OK, but I think put WKS in the picture one user is running app seems wrong.